rpc/swagger/swagger.yaml

```
@@ -830,6 +830,14 @@ paths:
             type: number
             default: 30
             example: 30
+        - in: query
+          name: order_by
+          description: Order by for sorting the transactions ("asc" (for ascending) || "desc" (for descending)). It sorts the transaction based on height and index.
+          required: false
+          schema:
+            type: number
```

```suggestion
            type: string
```

@marbar3778 ^ this is the sign we need to get RPC tests back